backend/corpora/api_server/app.py

```diff
@@ -19,8 +19,15 @@ def create_flask_app():
     connexion_app = connexion.FlaskApp(f"{APP_NAME}-{DEPLOYMENT_STAGE}", specification_dir="backend/config")
     # From https://github.com/zalando/connexion/issues/346
     connexion_app.app.url_map.strict_slashes = False
-    swagger_spec_path = f"{APP_NAME}.yml"
-    connexion_app.add_api(swagger_spec_path, validate_responses=True)
+    options = {"swagger_ui": True, 'swagger_url': '/'}
+    dataportal_api = super(connexion.FlaskApp, connexion_app).add_api(
+        f"{APP_NAME}.yml", validate_responses=True, options=options
+    )
+    curator_api = super(connexion.FlaskApp, connexion_app).add_api(
+        "curation-api.yml", validate_responses=True, options=options
```

Comment:
is there any reason to have the app come in as an env variable `APP_NAME` but not for the curation api? Should we hardcode both? 🤔